hoodie-client/src/test/java/com/uber/hoodie/table/TestCopyOnWriteTable.java

```diff
@@ -104,7 +105,7 @@ private HoodieWriteConfig makeHoodieClientConfig() throws Exception {
     private HoodieWriteConfig.Builder makeHoodieClientConfigBuilder() throws Exception {
         // Prepare the AvroParquetIO
         String schemaStr = IOUtils.toString(getClass().getResourceAsStream("/exampleSchema.txt"), "UTF-8");
-        return HoodieWriteConfig.newBuilder().withPath(basePath).withSchema(schemaStr);
+        return HoodieWriteConfig.newBuilder().withPath(basePath).withSchema(schemaStr).withWriteStatusClass(SampleMergeMetadataWriteStatus.class);
```

Comment:
Please write a separate test case, which demonstrates how the framework should be used. Lets not change the existing tests with a custom WriteStatus class, as I want to ensure the default usage with WriteStatus.class continues to be exercised.